Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -807,11 +828,53 @@ def get_indicator_results(
                               outputs_prefix='MISP.Attribute',
                               outputs_key_field='ID',
                               readable_output=readable_output)
+
     else:
+        if search_warninglists:
+            values = argToList(value)
+            res: list = []
+            misp_warninglists_response = PYMISP.values_in_warninglist(values)
+            if 'errors' in misp_warninglists_response:
+                raise DemistoException(
+                    f'Unable to validate against MISP warninglists!\nError message: {misp_warninglists_response}')
+            if misp_warninglists_response and len(misp_warninglists_response.items()) > 0:
+                for _, lists in misp_warninglists_response.items():
+                    list_names: str = ",".join([x["name"] for x in lists])
+                    dbot = Common.DBotScore(
+                        indicator=value,
+                        indicator_type=indicator_type,
+                        score=Common.DBotScore.GOOD, reliability=reliability,
+                        malicious_description=f"Match found in MISP warninglist{list_names}"
+                    )
+                    res.append(
+                        {
+                            "Value": value,
+                            "Count": len(lists),
+                            "Lists": ",".join([x["name"] for x in lists]),
+                        }
+                    )
+                    human_readable = tableToMarkdown(
+                        "MISP Warninglist matchings:",
+                        sorted(res, key=lambda x: x["Count"], reverse=True),
```

Comment:
why do we need sorted here? res is a list with one dictionary value

Reply:
Sorting is not necessary here but it helps to keep the result deterministic. I can remove it if you want to.

Reply:
res is a list of one dictionary so we can remove the sorting